pkg/cli/containers_test.go

```diff
@@ -2,17 +2,18 @@ package cli
 
 import (
 	"fmt"
+	"io"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
 	apiv1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
 	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
 	"github.com/acorn-io/acorn/pkg/client"
 	"github.com/acorn-io/acorn/pkg/mocks"
 	"github.com/golang/mock/gomock"
-	"io"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"os"
-	"strings"
-	"testing"
-	"time"
 
```

Comment:
nit: looks like there might be some places where we have 3 import groups instead of two

Reply:
good catch. thanks. fixed em